rero_ils/modules/documents/dumpers/indexer.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019-2022 RERO
+# Copyright (C) 2019-2022 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Indexing dumper."""
+from flask import current_app
+from invenio_records.dumpers import Dumper
+
+from ..extensions import TitleExtension
+from ..utils import process_literal_contributions
+
+
+class IndexerDumper(Dumper):
+    """Document indexer."""
+
+    @staticmethod
+    def process_holdings(record, data):
```

Comment:
seems private methods, isn't it ? ; best practice is to prefix the method name with a "_" (underscore).
Same for  `process_identifiers`